### PR TITLE
Ft days exceeding 30 #139551333

### PR DESCRIPTION
**Pull request by NgangaMaryanne**
#### What does this PR do?
Adds a feature where the user can set their nag time to more than 30 days but a maximum of 90 days.

#### Description of Task to be completed?
Add tests for timeout and grace time of more than 30 days.
Add functionality for the same.

#### How should this be manually tested?

Clone the repository using:
_git clone git@github.com:andela/healthchecks_akira.git_

Create virtual environment.

Pip install the requirements.txt file into your virtual environment
_pip install requirements.txt_

Cd into the healthchecks_akira directory
_cd healthchecks_akira_

Run the manage.py file
_python manage.py runserver_

#### Any background context you want to provide?
Previously user could only set timeout and grace period of not more than 30 days but now the user is able to set daily, weekly, monthly and 3-month jobs.

#### What are the relevant pivotal tracker stories?
Ft #139551333

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ __pycache__/
 .coverage
 hc.sqlite
 hc/local_settings.py
+hc/settings.py
```
Comment (osule) [inline .gitignore]: We shouldn't ignore settings.py. Personal configuration should be entered in hc/local_settings.py

**Review by osule — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (NgangaMaryanne) [reply .gitignore]: changed.

File: hc/settings.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 ALLOWED_HOSTS = [
     'healthchecks-akira.herokuapp.com',
 ]
-DEFAULT_FROM_EMAIL = 'healthchecks@example.org'
+DEFAULT_FROM_EMAIL = 'maryanne.nganga@andela.com'
```
Comment (osule) [inline hc/settings.py]: Please leave as is. This may be better for us as an environment variable. cc: @daisyndungu @Gathondu

Comment (Didah-A) [reply hc/settings.py]: done

**Review by osule — APPROVED** [review]
Looks great to me!

**Pull request merged**